docs/tutorials/feature-matrix.rst

```diff
@@ -3,27 +3,28 @@ Feature Matrix
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Gap/Filler                | yes      | yes            | yes      | read-only | yes        | skipped   | yes       |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Markers                   | yes      | yes            | yes      | planned   | planned    | no        | yes       |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Nesting                   | yes      | no             | yes      | read-only | write-only | flattened | yes       |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Transitions               | yes      | yes            | planned  | read-only | write-only | no        | planned   |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Audio/Video Effects       | planned  | needs research | planned  | planned   | planned    | no        | planned   |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Speed Effects             | yes      | yes            | planned  | read-only | planned    | no        | planned   |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
-|Color Decision List (CDL) | metadata | yes            | planned  | planned   | planned    | planned   | no        |
-+--------------------------+----------+----------------+----------+-----------+------------+-----------+-----------+
\ No newline at end of file
++--------------------------+----------+-------+--------+--------+-------+--------+---------+----+
+|Feature                   | OTIO   | EDL     |FCP7 XML| FCP X  | AAF   |   RV   | ALE     |Gst |
++==========================+==========+=======+========+========+=======+========+=========+====+
+|Single Track of Clips     |    ✔   |    ✔    |    ✔   |   ✔    |   ✔   | W-O    |    ✔    | ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Multiple Video Tracks     |    ✔   |    ✖    |    ✔   |   ✔    |   ✔   | W-O    |flattened| ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Audio Tracks & Clips      |    ✔   | testing |    ✔   |   ✔    |   ✔   | W-O    |    ✔    | ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Gap/Filler                |    ✔   |    ✔    |    ✔   |   ✔    |   ✔   |   ✔    |skipped  | ✔  |
++--------------------------+--------+---------+--------+--------+-------+--------+---------+----+
+|Markers                   |    ✔   |    ✔    |    ✔   |contrib?|planned|contrib?|   ✖     | ✔  |
```

Comment:
@eric-with-a-c do you know if the FCP-X adapter supports markers as well (I see it mentioned in the code)?  And RV adapter does not support markers, but there is an open issue for that (I think we should just put an X in that box).